src/index.ts

```diff
@@ -23,6 +26,11 @@ app.use(express.json());
 app.use(morgan('tiny'));
 app.use(express.static('public'));
 
+passport.initialize();

```

Comment:
Shouldn't it be `app.use(passport.initialize())`? If it works now then ignore my comment 👍🏻